src/workflow/ArcanistLandWorkflow.php

```diff
@@ -1665,18 +1564,17 @@ private function checkForBuildablesWithPlanBehaviors($diff_phid) {
         ),
       ));
 
-    if (!$raw_buildables['data']) {
+    if (!$raw_buildable['data']) {
       return;
     }
 
-    $buildables = $raw_buildables['data'];
-    $buildable_phids = ipull($buildables, 'phid');
-
+    $buildable = head($raw_buildable['data']);
```

Comment:
Do we know the case when diff_phid is associated with more then one Buildable?

Reply:
No, but that should not be the case -- there should only ever be one Buildable as we rely on that for overall status of passing vs failing checks